README.md

```diff
@@ -1,13 +1,94 @@
-debos
-=====
+# debos -  Debian OS images builder
 
-Debian OS builder. debos is a tool to make creation of various debian based os
-"images" simpler. While most other tools focus on specific use-case, debos is
-more meant as a toolchain to make comon actions trivial while providing enough
-rope to do whatever tweaking that might be required behind the scene.
+## Sypnosis
 
-debos expects a yaml file as input, syntax description can be found at:
-  https://godoc.org/github.com/go-debos/debos/actions
+    debos [options] <recipe file in YAML>
+    debos [--help]
+
+Application Options:
+
+          --artifactdir=
+      -t, --template-var=   Template variables
+          --debug-shell     Fall into interactive shell on error
+      -s, --shell=          Redefine interactive shell binary (default: bash)
+          --scratchsize=    Size of disk backed scratch space
+
+
+## Description
+
+debos is a tool to make the creation of various Debian-based OS images
+simpler. While most other tools focus on specific use-cases, debos is
+more meant as a tool-chain to make common actions trivial while providing
+enough rope to do whatever tweaking that might be required behind the scene.
+
+debos expects a YAML file as input and will run the actions listed in the
+file sequentially. These actions should be self-contained and independent
+of each other.
+
+Some of the actions provided by debos to customize and produce images are:
+
+* apt: install packages and their dependencies with 'apt'
+* debootstrap: construct the target rootfs with debootstrap
+* download: download a single file from the internet
+* FilesystemDeploy: deploy a root filesystem to an image previously created
```

Comment:
I'm using the names listed at https://godoc.org/github.com/go-debos/debos/actions
Although I  missed the capitalization in the first three.

Reply:
The documentation there says they're spelled like
```
- action: filesystem-deploy
  setup-fstab: bool
  setup-kernel-cmdline: bool
```
in the YAML, which I think is more user-facing than the Go code? (Perhaps the CamelCase names also work, I haven't tried...)

Reply:
I mean in the headers. For example for FileSystemDeploy: https://godoc.org/github.com/go-debos/debos/actions#hdr-FilesystemDeploy_Action   "FilesystemDeploy Action"
I don't mind using the YAML instead of the go functions name as long as we try to be coherent in both documents!